src/components/LoanCards/Buttons/LendCtaExp.vue

```diff
@@ -173,6 +180,12 @@ export default {
 				selectedDollarAmount
 			);
 		},
+		getDropdownERL() {
+			if (this.activeCookie !== '') {
+				return getCookieDropdown(this.activeCookie, this.userBalance, this.unreservedAmount);
+			}
+			return '25';
+		}
 	},
 	watch: {
 		unreservedAmount(newValue, previousValue) {
```

Comment:
Does the `unreservedAmount` `watch` need the same checks that the other CTA component has?